daemon/cluster/swarm.go

```diff
@@ -430,6 +430,14 @@ func (c *Cluster) Info() types.Info {
 		info.NodeID = state.swarmNode.NodeID()
 	}
 
+	clusterInfo.WithValues(
+		info.Cluster.ID,
+		info.NodeID,
+		info.NodeAddr,
+	).Set(1)
+	clusterManagers.Set(float64(info.Managers))
+	clusterNodes.Set(float64(info.Nodes))
```

Comment:
Indeed, that's perfect! It would be better if `go-metrics` had an interface for it. But here is how @crosbymichael used it: https://github.com/crosbymichael/cgroups/pull/8/files. I'll give it a try!